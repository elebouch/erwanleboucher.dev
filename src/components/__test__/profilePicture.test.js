import React from "react"
import { render } from "@testing-library/react"
import "jest-styled-components"

import ProfilePicture from "../profilePicture"

describe("ProfilePicture", () => {
  it("renders correctly", () => {
    const { container } = render(<ProfilePicture />)
    expect(container.firstChild).toMatchSnapshot()
  })
})
