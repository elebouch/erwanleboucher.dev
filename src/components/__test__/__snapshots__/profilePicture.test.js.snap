// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ProfilePicture renders correctly 1`] = `"<div class=\\"sc-AxjAm Oxgzp gatsby-image-wrapper\\" style=\\"position: relative; overflow: hidden; display: inline-block;\\"><picture><source><img alt=\\"\\" loading=\\"lazy\\" style=\\"position: absolute; top: 0px; left: 0px; width: 100%; height: 100%; object-fit: cover; object-position: center; opacity: 0; transition: opacity 500ms;\\"></picture><noscript><picture><source srcset=\\"undefined\\" /><img loading=\\"lazy\\" src=\\"\\" alt=\\"\\" style=\\"position:absolute;top:0;left:0;opacity:1;width:100%;height:100%;object-fit:cover;object-position:center\\"/></picture></noscript></div>"`;
